### DISCUSSION TITLE
Edit / Customize parts of the standard interface

### DISCUSSION THREAD

**Discussion by felipe-dr** [decaporg/decap-cms#6888] [Q&A] [answered]
Good morning everything is fine?

**1 )**
What is the recommended way to edit / customize the text of parts of the standard interface. After all, is this possible?
Here are some examples below

![interface-1](https://github.com/decaporg/decap-cms/assets/62888625/2e0083cb-a104-43af-88bc-6fdbcdfe2154)

![interface-2](https://github.com/decaporg/decap-cms/assets/62888625/67324cd2-3ac0-4829-9dbc-a343e7c26bf1)

![interface-3](https://github.com/decaporg/decap-cms/assets/62888625/917f9465-9b3d-4dbc-b230-6b5689a04ebf)

![interface-4](https://github.com/decaporg/decap-cms/assets/62888625/be7fc4a0-9148-4159-a8f6-317d88cf68dc)

**2 )**
What is the recommended way to edit / customize field validation messages, as well as messages displayed at the top of the screen (toast)

![interface-5](https://github.com/decaporg/decap-cms/assets/62888625/12638d6d-7e35-47a5-8720-667483332ef9)

I know there is a beta feature "i18n Support" however, there is no support for PT-BR (Brazilian Portuguese) language. Besides that it would only translate parts of collections and not the interface as pointed out above.

Thanks.

Felipe D.R.

**Comment by martinjagodic** [comment]
@felipe-dr locales are stored in the package [`decap-cms-locales`](https://github.com/decaporg/decap-cms/tree/master/packages/decap-cms-locales). We would happily accept a contribution for PT-BR.

**Comment by felipe-dr** [comment]
Hi Martin, thanks

As you pointed out, I was able to locate a folder called "pt" and from what I compared with the "en" folder they have all the same properties, so does the language already exist ready for use?

If you need me to continue or act in any specific PR for the translation, I will be available and you can let me know.

**obs:** if the pt folder is for Portuguese from Portugal, even so, I can tell you that all phrases are the same if they are written in Portuguese BR. But if you think it's better to create a specific folder for PT-BR, I'd be grateful if I could open a PR too.

Thanks.

Felipe D.R.

**Reply by martinjagodic** [reply, marked as answer]
If there is no difference, there is no need to add pt-br. 

It is explained [in the docs](https://decapcms.org/docs/configuration-options/#locale) how you can set UI locale.

TLDR: set `locale: 'pt'` in your config.yaml
